app/models/hmpps_api/offender_base.rb

```diff
@@ -13,7 +13,9 @@ class OffenderBase
              :parole_review_date, :victim_liaison_officers, :updated_at,
              to: :@case_information, allow_nil: true
 
-    attr_accessor :category_code, :date_of_birth, :prison_arrival_date, :sentence, :allocated_pom_name
+    delegate :code, :label, :active_since, to: :@category, prefix: :category, allow_nil: true
```

Comment:
not sure about the allow_nil - surely category will always be filled in?

Reply:
It looks like offenders don't _always_ have a category. If they're new in prison and an offender categorisation assessment hasn't been completed yet, then their category will legitimately be `nil`.

I've added a test to `offender_spec.rb` to cover this scenario, to make it clear that this is expected behaviour.